examples/requirements-and-tests/package.json

```diff
@@ -0,0 +1,74 @@
+{
+    "name": "requirements-and-tests",
+    "displayName": "requirements-and-tests",
+    "description": "Please enter a brief description here",
+    "version": "0.0.1",
+    "engines": {
+        "vscode": "^1.56.0"
+    },
+    "categories": [
+        "Programming Languages"
+    ],
+    "contributes": {
+        "languages": [{
+            "id": "requirements-language",
+            "aliases": ["Requirements Language", "requirements-language"],
+            "extensions": [".req"],
+            "configuration": "./language-configuration.json"
+        },{
+            "id": "tests-language",
+            "aliases": ["Tests Language", "tests-language"],
+            "extensions": [".tst"],
+            "configuration": "./language-configuration.json"
+        }],
+        "grammars": [{
+            "language": "requirements-language",
+            "scopeName": "source.requirements-language",
+            "path": "./syntaxes/requirements.tmLanguage.json"
+        },{
+            "language": "tests-language",
+            "scopeName": "source.tests-language",
+            "path": "./syntaxes/tests.tmLanguage.json"
+        }]
+    },
+    "activationEvents": [
+        "onLanguage:requirements-and-tests-language",
+        "onLanguage:requirements-language",
+        "onLanguage:tests-language"
+    ],
+    "files": [
+        "bin",
+        "out",
+        "src"
+    ],
+    "bin": {
+        "requirements-and-tests-language-cli": "./bin/cli"
+    },
+    "main": "./out/extension.js",
+    "scripts": {
+        "vscode:prepublish": "npm run build && npm run lint",
+        "build": "tsc -b tsconfig.json",
+        "watch": "tsc -b tsconfig.json --watch",
+        "lint": "eslint src --ext ts",
+        "langium:generate": "langium generate",
+        "langium:watch": "langium generate --watch"
+    },
+    "dependencies": {
+        "chevrotain": "^9.1.0",
+        "chalk": "^4.1.2",
+        "commander": "^8.0.0",
+        "langium": "^0.4.0",
+        "vscode-languageclient": "^8.0.1",
+        "vscode-languageserver": "^8.0.1",
+        "vscode-uri": "^3.0.2"
+    },
+    "devDependencies": {
+        "@types/node": "^14.17.3",
+        "@types/vscode": "^1.56.0",
+        "@typescript-eslint/eslint-plugin": "^5.28.0",
+        "@typescript-eslint/parser": "^5.28.0",
+        "eslint": "^8.17.0",
+        "langium-cli": "^0.4.0",
+        "typescript": "^4.7.4"
```

Comment:
Some of these dependencies are already declared in the root package.json, please remove those from here to reduce maintenance effort.